client/src/main/scala/com.gu.contentapi.client/model/Queries.scala

```diff
@@ -241,6 +241,7 @@ trait FilterParameters[Owner <: Parameters[Owner]] extends Parameters[Owner] { t
 trait FilterExtendedParameters[Owner <: Parameters[Owner]] extends Parameters[Owner] { this: Owner =>
   def tag = StringParameter("tag")
   def ids = StringParameter("ids")
+  def paths: StringParameter = StringParameter("paths")
```

Comment:
Apologies for the pedantry, but is there any reason why this one should have an explicitly type annotation when the other methods do not?

Reply:
Oh. Probably not - I was likely just trying to stop IntelliJ from nagging at me and then became blind to it. I can fix that now (I have a test and everything!)

Reply:
Yes, see... 
<img width="733" alt="Screenshot 2021-03-19 at 17 06 02" src="https://user-images.githubusercontent.com/690395/111817461-ee28e080-88d5-11eb-9284-c80444d0912f.png">

Reply:
Fixed now

Reply:
Thought it was probably that.  Would be nice if IntelliJ could turn off that rule when it is already so clear what the return type is. Thanks for fixing it.  And sorry for pointing out such a minor thing.

Reply:
No worries. Thanks for spotting it!